### DISCUSSION TITLE
Query: PROD start script?

### DISCUSSION THREAD

**Discussion by Shadowfita** [docmost/docmost#356] [Help / Questions] [answered]
Does anyone know which startup scripts are used in production by the main docker image? I'm hoping to pull a forked repo down directly and run it in a prod environment. I'm assuming a combination of: 

```
pnpm nx run build
pnpm nx run start:prod
```

I can't see how the client is run also in this format though.

**Comment by Philipinho** [comment, marked as answer]
You have to build the react client app. We serve the built client on the server with `serve-static`.

Using `pnpm build` on the root directory builds all the necessary apps and packages.

See the build process in the Dockerfile https://github.com/docmost/docmost/blob/main/Dockerfile.

**Reply by Shadowfita** [reply]
Thank you very much @Philipinho  !
